src/controller/user-controller.ts

```diff
@@ -1610,14 +1615,13 @@ export default class UserController extends BaseController {
    */
   public async waiveUserFines(req: RequestWithToken, res: Response): Promise<void> {
     const { id: rawId } = req.params;
-    this.logger.trace('Waive fines of user', rawId, 'by', req.token.user);
```

Comment:
This removes the ability to waive all fines in a single call right?

Reply:
No, this is still possible.
- As part of the transition to this new format, not sending any body to the endpoint will behave the same as it does now, namely waive all fines.
- When you give the total amount of fines as amount to waive, all fines will be waived.